packages/cosmwasm-stargate/src/cosmwasmclient.ts

```diff
@@ -334,10 +334,24 @@ export class CosmWasmClient {
     return codeDetails;
   }
 
+  /**
+   * getContracts() returns all contracts and is just looping through all pagination pages.
+   *
+   * This is potentially inefficient and advanced apps should consider creating
+   * their own query client to handle pagination together with the app's screens.
+   */
   public async getContracts(codeId: number): Promise<readonly string[]> {
-    // TODO: handle pagination - accept as arg or auto-loop
-    const { contracts } = await this.forceGetQueryClient().wasm.listContractsByCodeId(codeId);
-    return contracts;
+    const allContracts = [];
+    let startAtKey: Uint8Array | undefined = undefined;
+    do {
+      const { contracts, pagination }: QueryContractsByCodeResponse =
+        await this.forceGetQueryClient().wasm.listContractsByCodeId(codeId, startAtKey);
```

Comment:
Nope - `'pagination' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.ts`

Reply:
Okay, I can reproduce it but I don't understand what the error means and why we get it. Anyways, let's move on.